### ISSUE TITLE
Docker installation 'mix omg.server' port already in use

### ISSUE THREAD

**Issue by vanmil** [omisego/ewallet#399]
# Problem
During the installation (https://github.com/omisego/ewallet/blob/master/docs/setup/docker.md) I received the following error that the port 4000 is already in use. 
```
[debug] [:nonode@nohost][Elixir.Quantum.ExecutionBroadcaster] Adding job :expire_consumptions
[debug] [:nonode@nohost][Elixir.Quantum.ExecutionBroadcaster] Adding job :expire_requests
[debug] [:nonode@nohost][Elixir.Quantum.ExecutionBroadcaster] Continuing Execution Broadcasting at -576460719765 (2018-08-01T19:47:00)
[info] Setting up websockets dispatchers...
[error] Could not start watcher "yarn" from "/app/apps/admin_panel/assets", executable does not exist
[info] Running UrlDispatcher.Plug with Cowboy http on port 4000
[error] Failed to start Ranch listener UrlDispatcher.Plug.HTTP in :ranch_tcp:listen([port: 4000]) for reason :eaddrinuse (address already in use)

** (Mix) Could not start application url_dispatcher: UrlDispatcher.Application.start(:normal, []) returned an error: shutdown: failed to start child: {:ranch_listener_sup, UrlDispatcher.Plug.HTTP}
    ** (EXIT) shutdown: failed to start child: :ranch_acceptors_sup
        ** (EXIT) {:listen_error, UrlDispatcher.Plug.HTTP, :eaddrinuse}
[info] Application url_dispatcher exited: UrlDispatcher.Application.start(:normal, []) returned an error: shutdown: failed to start child: {:ranch_listener_sup, UrlDispatcher.Plug.HTTP}
    ** (EXIT) shutdown: failed to start child: :ranch_acceptors_sup
        ** (EXIT) {:listen_error, UrlDispatcher.Plug.HTTP, :eaddrinuse}
```

I experimented with different ports, getting the same error. I also checked if the port was already in use on my macbook, which it isn't (but `docker ps` lists the `omisego/ewallet:latest` image as using port 4000/tcp.

# Environment
OS: mac osx 10.13.5
Docker: 18.03
Docker-compose: 1.21.1

# Steps to reproduce
Follow the installation instructions on https://github.com/omisego/ewallet/blob/master/docs/setup/docker.md. The problem arrises step 3.

**Comment by jarindr** [comment]
Hello @vanmil thank you for reporting this issue, we will look into it.

**Comment by vanmil** [comment]
I figured out that adding `ports: - 4000:4000` to the `docker-compose.yml` solves this problem. 

So when using the following it should work:
```
version: "3"
services:
  postgres:
    image: postgres:9.6.9-alpine
    restart: always
    volumes:
      - postgres-db:/var/lib/postgresql/data
    networks:
      - internal
    environment:
      POSTGRESQL_PASSWORD: passw0rd
    healthcheck:
      test: ["CMD-SHELL", "pg_isready", "-U", "postgres"]
      interval: 30s
      retries: 3

  mailhog:
    image: mailhog/mailhog:v1.0.0
    restart: always
    networks:
      - internal

  ewallet:
    image: omisego/ewallet:latest
    restart: always
    networks:
      - internal
      - external
    depends_on:
      - postgres
    environment:
      DATABASE_URL: "postgresql://postgres:passw0rd@postgres:5432/ewallet"
      LOCAL_LEDGER_DATABASE_URL: "postgresql://postgres:passw0rd@postgres:5432/local_ledger"
      EWALLET_SECRET_KEY: "<ewallet_secret_key_here>"
      LOCAL_LEDGER_SECRET_KEY: "<local_ledger_secret_key_here>"
      SMTP_HOST: mailhog
      SMTP_PORT: 1025
    ports:
      - 4000:4000

networks:
  external:
  internal:

volumes:
  postgres-db:
```

**Comment by T-Dnzt** [comment]
@vanmil Could you submit a PR with your fix? Thank you!

**Comment by unnawut** [comment]
Closed by #405

**Comment by unnawut** [comment]
Sorry. Got confused. The PR is still building...
